releases/mainnet/1.2.9/index.html

```diff
@@ -0,0 +1,24 @@
+<html>
+	<head>
+		<title>Mainnet Release 1.2.9</title>
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Mainnet Release 1.2.9</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.2.9/contracts.json">Contract addresses</a></li>
+				<li><a href="">Contract addresses (IPFS hosted)</a></li>
```

Comment:
Same here. Can IPFS link be added?

Reply:
This will be done as soon as the spell is cast. I don't think we should rush and have an incomplete version posted before as anyway this collateral type is not open to the public. After the spell is cast, we are going to add the missing address for mainnet, create the IPFS files (kovan and mainnet) and change the active symlinks. Then we could merge it.

Reply:
Done